src/features/create-token/CreateTokenBody.tsx

```diff
@@ -0,0 +1,71 @@
+//- React Imports
+import React, { FC } from 'react';
+
+//- Component Imports
+import { DetailsForm } from './DetailsForm';
+import { TokenomicsForm } from './TokenomicsForm/TokenomicsForm';
+import { TokenSummary } from './TokenSummary/TokenSummary';
+
+//- Type Imports
+import { DetailsFormSubmit, TokenomicsFormSubmit } from './CreateToken.types';
+import { MediaType } from '@zero-tech/zui/src/components/MediaInput';
+
+type CreateTokenBodyProps = {
+	stepId: string;
+	detailsFormValues: DetailsFormSubmit;
+	onDetailsSubmit: (values: DetailsFormSubmit) => void;
+	tokenomicsFormValues: TokenomicsFormSubmit;
+	onTokenomicsSubmit: (values: TokenomicsFormSubmit) => void;
+	onMediaInputChange: (
+		mediaType: MediaType,
+		previewUrl: string,
+		image: Buffer,
+	) => void;
+	onLaunchSubmit: () => void;
+	onClose: () => void;
+};
```

Comment:
Think I mentioned in the last PR about moving this to Context API, but I won't make it a requirement as this will work perfectly.